components/permissions/android/java/src/org/chromium/components/permissions/BravePermissionDialogDelegate.java

```diff
@@ -49,4 +88,174 @@ public void setSelectedLifetimeOption(int idx) {
     public int getSelectedLifetimeOption() {
         return mSelectedLifetimeOption;
     }
+
+    public boolean getUseWalletLayout() {
+        return mUseWalletLayout;
+    }
+
+    @CalledByNative
+    public void setUseWalletLayout(boolean useWalletLayout) {
```

Comment:
adding individual setters/getters for wallet properties on BravePermissionDialogDelegate doesn't seem right to me. If this is a completely different prompt then it seems like we should be subclassing `PermissionDialogController` and calling `Java_EtheruemPermissionDialogController_createDialog`

Reply:
that code is removed and we have a permission dialog in a browser process now